WeakAuras/TSUHelpers.lua

```diff
@@ -43,30 +43,52 @@ local removeAll = function(states)
   return changed
 end
 
-local function recurseUpdate(t1, t2)
+local skipKeys = {
```

Comment:
i don't quite understand the point of this?

Reply:
It fixes the bug with

```Lua
function(states, event, updatedTriggerNumber, state)
    if event == "TRIGGER" and state then
        states:Update("", state[""])
    end
end
```

copying state[""].trigger into states[""].trigger which resulted in WeakAurasOptions replacing the custom trigger view with a copy of the view of an other trigger, cf https://discord.com/channels/172440238717665280/598903823800467496/1360233139452121088